### DISCUSSION TITLE
Using the PMOD I2S2 with the pico

### DISCUSSION THREAD

**Discussion by SnazzyBlanket** [pschatzmann/arduino-audio-tools#1408] [Q&A] [answered]
https://digilent.com/reference/pmod/pmodi2s2/reference-manual

i am wondering if it would be possible to connect this to an audio stream with the Pico using arduino audio tools (as it has support for both input and output it would be a good peripheral to use)
thanks

**Comment by pschatzmann** [comment, marked as answer]
I never tried, but theoretically it should work. 
If I remember right Andreas Spiess used this module in his video...

**Reply by SnazzyBlanket** [reply]
Yes exactly that is why i Bought one years ago as I followed along and got all of his videos working during Covid. kept me from losing my mind
